Makefile

```diff
@@ -51,8 +53,15 @@ examples_install:
 	${MKDIR} ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 	${INSTALL} -m 0644 examples/dnsdoctor/*.lua ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 
+.PHONY: examples
+examples:
+	${MAKE} -C examples/filter
+	${MKDIR} ${LUNATIK_EBPF_INSTALL_PATH}
+	${INSTALL} -m 0644 examples/filter/https.o ${LUNATIK_EBPF_INSTALL_PATH}/
```

Comment:
can you move these two lines to `examples_install:`.. I think it makes more sense.. I would leave only `${MAKE} -C examples/filter` in `examples:`

Reply:
sorry, actually.. this may broke the installation of other examples if we don't build ebpf in the target.. my suggestion is to create two new rules here.. `ebpf:` and `ebpf_install:`, e.g.,

```
.PHONY:  ebpf
ebpf:
	${MAKE} -C examples/filter
	
ebpf_install:
	${MKDIR} ${LUNATIK_EBPF_INSTALL_PATH}
	${INSTALL} -m 0644 examples/filter/https.o ${LUNATIK_EBPF_INSTALL_PATH}/
```

I would also update the [documentation](https://github.com/luainkernel/lunatik#usage-7) with the new steps..

```
sudo make examples_install                   # installs examples
make ebpf                                         # builds the XDP/eBPF program
sudo make ebpf_install                   # installs the XDP/eBPF program
sudo lunatik run examples/filter/sni false   # runs the Lua kernel script
sudo xdp-loader load -m skb <ifname> https.o # loads the XDP/eBPF program
```